setup.cfg

```diff
@@ -134,7 +134,8 @@ filterwarnings =
     ignore:Distutils was imported before Setuptools
     # Figure tests use agg backend so show doesn't work but we use it in peek
     ignore:Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.
-    # We raise this for our HECClient to avoid HTTPS issues with non https urls.
+    # Currently helio makes unverified requests - this filter should be removed when
+    # https://github.com/sunpy/sunpy/issues/4401 is fixed
     ignore:Unverified HTTPS request
```

Comment:
We should not be ignoring this globally, if another server starts making unverified https requests we should be erroring. I think you should revert this change back to a mark in the helio tests.

Reply:
Fixed.